torch/csrc/jit/interpreter.cpp

```diff
@@ -408,6 +414,15 @@ struct CodeImpl {
     return constant_table_;
   }
 
+  void request_bailout(size_t index) {
+    size_t instr_index = bi_to_ii_[index];
```

Comment:
use `.at` to avoid segfaults when this is called incorrectly

Reply:
note: it might be simpler to just linearly scan through to the nth GUARD node in the graph rather than maintaining a map. request_bailout is not perf sensitive, and it would require less machinery.